packages/azure_cli.rb

```diff
@@ -1,35 +1,29 @@
+# This package is left untouched, except upgrades, as many changes are in #5503
+
 require 'package'
 
 class Azure_cli < Package
   description 'The Azure CLI 2.0 is Azure\'s new command-line experience for managing Azure resources.'
   homepage 'https://github.com/Azure/azure-cli'
   version '2.21.0'
   license 'MIT'
-  compatibility 'i686 x86_64'
+  compatibility 'i686,x86_64'
   case ARCH
   when 'i686', 'x86_64'
-    source_url 'https://github.com/Azure/azure-cli/archive/refs/tags/azure-cli-2.21.0.tar.gz'
+    source_url 'https://github.com/Azure/azure-cli/archive/azure-cli-2.21.0.tar.gz'
     source_sha256 'bbe4a1f85418d239444717f2c9706a87f81fd2515bb0bb4b4e48548fd3e08caa'
-
-    binary_url({
-      i686: 'https://dl.bintray.com/chromebrew/chromebrew/azure_cli-2.21.0-chromeos-i686.tar.xz',
-    x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/azure_cli-2.21.0-chromeos-x86_64.tar.xz'
-    })
-    binary_sha256({
-      i686: '222e74caa74df6ab129f33f90aea86e65d000ebc4f5d78cb7f3ec2f98421ed3c',
-    x86_64: '073b64f7104e298205afe623588c95f2e0cb17e664e53ff46042c9bd4076d2c8'
-    })
   end
 
   def self.install
-    system "pip install --prefix #{CREW_PREFIX} --root #{CREW_DEST_DIR} -I azure-cli==#{version} --no-warn-script-location"
+    system "pip install azure-cli -r requirements.txt --root #{CREW_DEST_DIR} --prefix #{CREW_PREFIX}"
```

Comment:
Don't you want to use the local source instead?  If you do it this way, you should keep `-I azure-cli==#{version}`.

Reply:
I'm going to unpipify this package like all the other packages since #5503 is merged.